lib/unsupported-features/node-builtins-modules/module.js

```diff
@@ -18,6 +18,7 @@ const Module = {
         },
     },
     enableCompileCache: { [READ]: { experimental: ["22.8.0"] } },
+    flushCompileCache: { [CALL]: { experimental: ["23.0.0", "22.10.0"] } },
```

Comment:
I see `enableCompileCache`, `getCompileCacheDir` and other methods use `READ`, but since they're function to be called they should all use `CALL` I presume?

Hence why I used `CALL` instead of `READ` here as well

Reply:
```js
module.enableCompileCache;
module.enableCompileCache();
```

* using `[READ]` - it report all the 2 cases;
* using `[CALL]` - it report the 2rd case;

seems reasonable to use `CALL`.  thoughts?  /@scagood

Reply:
I just mark everything as `READ`, as we're checking if it's there or not. :thinking: 

I am happy to change that you both think `CALL` is more reasonable.

```js
module.enableCompileCache; // Not matched
module.enableCompileCache(); // Matched

// I have not tested, I assume this does not match
module.enableCompileCache.call(module); // Not matched ?
```

Reply:
```js
module.enableCompileCache.call(module); // Not matched ?
```

yes, I see, and confirmed it's not matched.

just opened https://github.com/eslint-community/eslint-plugin-n/issues/359 for discussion.  For now, for consistency, let's just use `[READ]` - we can change all these to `[CALL]` once reached consensus. 😄